File: tuts/004-cloudmap-custom-attributes/go/scenarios/getting_started.go
// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
// SPDX-License-Identifier: Apache-2.0

package main

import (
	"context"
	"fmt"
	"github.com/aws/aws-sdk-go-v2/config"
	aws_servicediscovery "github.com/aws/aws-sdk-go-v2/service/servicediscovery"
)

func main() {
	cfg, _ := config.LoadDefaultConfig(context.TODO())
	client := aws_servicediscovery.NewFromConfig(cfg)
	_ = client
	fmt.Println("Running CloudMap getting started scenario...")
	// TODO: setup, interact, teardown
}

File: tuts/004-cloudmap-custom-attributes/javascript/scenarios/getting-started.js
// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
// SPDX-License-Identifier: Apache-2.0

import { ServiceDiscoveryClient } from "@aws-sdk/client-servicediscovery";
import { randomUUID } from "crypto";

const main = async () => {
  const client = new ServiceDiscoveryClient({});
  const suffix = randomUUID().slice(0, 8);
  console.log("Running servicediscovery getting started scenario...");
  // TODO: implement setup, interact, teardown
  console.log("Scenario complete.");
};

main();
